luatest/init.lua

```diff
@@ -38,6 +38,9 @@ local parametrizer = require('luatest.parametrizer')
 --
 luatest.log = require('luatest.log')
 
+-- Simple Tarantool runner and output catcher
```

Comment:
To be consistent with neighbor code, please change:

`--` -> `---`
Add a dot to the end of the sentence.

Reply:
I'm against mimicry of ldoc comment style without actual usage of ldoc.

Reply:
Actually, we use it and publish docs to tarantool.io. Moreover, we have a [workflow](https://github.com/tarantool/luatest/actions/workflows/ldoc.yml) to test ldoc build.

Reply:
Wow, nice. I overlooked it.

So, my nits regarding ldoc's syntax below are applicable. We should verify how it is rendered.

Reply:
Fixed. See my comment below to see how it looks in the documentation.